src/shared/assets/fakePhotos.js

```diff
@@ -0,0 +1,69 @@
+const fakePhotos = [
```

Comment:
Is this file duplicated? `fakePhotos.js` is already in `/src/components/dummyData/`

Reply:
@GTOnizuka13 test ping